{
  "_from": "@fastify/error@^3.0.0",
  "_id": "@fastify/error@3.0.0",
  "_inBundle": false,
  "_integrity": "sha512-dPRyT40GiHRzSCll3/Jn2nPe25+E1VXc9tDwRAIKwFCxd5Np5wzgz1tmooWG3sV0qKgrBibihVoCna2ru4SEFg==",
  "_location": "/@fastify/error",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "@fastify/error@^3.0.0",
    "name": "@fastify/error",
    "escapedName": "@fastify%2ferror",
    "scope": "@fastify",
    "rawSpec": "^3.0.0",
    "saveSpec": null,
    "fetchSpec": "^3.0.0"
  },
  "_requiredBy": [
    "/fastify"
  ],
  "_resolved": "https://registry.npmjs.org/@fastify/error/-/error-3.0.0.tgz",
  "_shasum": "bfcb7b33cec0196413083a91ef2edc7b2c88455b",
  "_spec": "@fastify/error@^3.0.0",
  "_where": "/home/antonel/work/financeProject/BE-finance/node_modules/fastify",
  "author": {
    "name": "Tomas Della Vedova"
  },
  "bugs": {
    "url": "https://github.com/fastify/fastify-error/issues"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "A small utility, used by Fastify itself, for generating consistent error objects across your codebase and plugins.",
  "devDependencies": {
    "@types/node": "^17.0.0",
    "ava": "^4.0.1",
    "standard": "^17.0.0",
    "tsd": "^0.20.0",
    "typescript": "^4.1.3"
  },
  "homepage": "https://github.com/fastify/fastify-error#readme",
  "keywords": [
    "fastify",
    "error",
    "utility",
    "plugin"
  ],
  "license": "MIT",
  "main": "index.js",
  "name": "@fastify/error",
  "publishConfig": {
    "access": "public"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/fastify/fastify-error.git"
  },
  "scripts": {
    "test": "standard && ava -v && tsd"
  },
  "tsd": {
    "compilerOptions": {
      "esModuleInterop": true
    }
  },
  "types": "index.d.ts",
  "version": "3.0.0"
}
